src/main/java/org/codeforamerica/shiba/Money.java

```diff
@@ -0,0 +1,48 @@
+package org.codeforamerica.shiba;
+
+import java.io.Serial;
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+
+public class Money extends BigDecimal {
```

Comment:
I really like that you're encapsulating all of this in a Money class that we can reuse. That's a really good call